.werft/installer-tests.ts

```diff
@@ -144,8 +174,9 @@ function getKubeconfig() {
 export async function installerTests(config: TestConfig) {
     console.log(config.DESCRIPTION);
     for (let phase of config.PHASES) {
-        const phaseSteps = INFRA_PHASES[phase];
-        const ret = callMakeTargets(phaseSteps.phase, phaseSteps.description, phaseSteps.makeTarget);
+        const args = phase.split(" ");
+        const phaseSteps = INFRA_PHASES[args[0]];
+        const ret = callMakeTargets(phaseSteps.phase, phaseSteps.description, phaseSteps.makeTarget, args.slice(1));
```

Comment:
The way that we're splitting the phase to generate args, and then dropping the first element of the args is a little bit confusing. As we grow these tests it might be useful to extract an `args` list for phases that need arguments.

No need to change, but raising this as a potential point of friction in a few weeks/months.

Reply:
Updated this!